qa/rpc-tests/initial_reissuance_token.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2016 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#
+# Test chain initialisation when specifying number of reissuance tokens to issue.
+#
+
+from decimal import Decimal
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.authproxy import JSONRPCException
+from test_framework.util import *
+
+class InitialReissuanceTokenTest(BitcoinTestFramework):
+    """
+    Test creation of initial reissuance token for default asset on chain set up
+
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.setup_clean_chain = True
+        self.num_nodes = 2
+
+        #Set number of initial reissuance tokens and also set initial free coins less than max so we can reissue more later
+        self.node_args = [["-initialreissuancetokens=200000000", "-initialfreecoins=2000000000000000"], ["-initialreissuancetokens=200000000", "-initialfreecoins=2000000000000000"]]
+
+    def setup_network(self, split=False):
+        self.nodes = start_nodes(self.num_nodes, self.options.tmpdir, self.node_args)
+        connect_nodes_bi(self.nodes,0,1)
+        self.is_network_split = False
+        self.sync_all()
+
+    def run_test(self):
+        #Claim all anyone-can-spend 'bitcoin'
+        self.nodes[0].sendtoaddress(self.nodes[0].getnewaddress(), 20000000, "", "", True)
+        self.nodes[0].generate(101)
+        self.sync_all()
+
+        walletinfo=self.nodes[0].getwalletinfo()
+        balance=walletinfo['balance']
+        token=""
+
+        #Get the hex of the reissuance token
+        for i in balance:
+            print(i)
```

Comment:
no need for print here

Reply:
Ah yeah, that was me testing - forgot to remove. Shall I change?